<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NumberConv</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 127);
color: rgb(255, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="ccGBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="twoRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2 cc</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="sixtRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>16 cc</string>
     </property>
    </widget>
    <widget class="QLabel" name="ccLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>CC</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="eightRadio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>8 cc</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="fontGBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="boldBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bold</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="italicBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Italic</string>
     </property>
    </widget>
    <widget class="QLabel" name="fontLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="sizeGBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSlider" name="sizeSlider">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>6</number>
     </property>
     <property name="maximum">
      <number>24</number>
     </property>
     <property name="pageStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="sizeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="helpLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input your number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
